src/physics/cam/rk_stratiform_cam.F90

```diff
@@ -22,10 +22,10 @@ module rk_stratiform
 private
 save
 
-public :: rk_stratiform_register, rk_stratiform_init_cnst, rk_stratiform_implements_cnst
-public :: rk_stratiform_init
-public :: rk_stratiform_tend
-public :: rk_stratiform_readnl
+public :: rk_stratiform_cam_register, rk_stratiform_cam_init_cnst, rk_stratiform_cam_implements_cnst
```

Comment:
@jtruesdal it looks like these two subroutines and the module name are renamed in the latest CAM tag. I have pushed the changes to your branch.